distribution/docs/src/main/resources/_contents/_queries/filter-intro-contents.adoc

```diff
@@ -0,0 +1,94 @@
+
+=== Filters
+
+An OGC Filter is a http://www.opengeospatial.org/standards/filter[Open Geospatial Consortium (OGC) standard] that describes a query expression in terms of Extensible Markup Language (XML) and key-value pairs (KVP). 
+The ${ddf-catalog} Framework does not use the XML representation of the OGC Filter standard. ${branding} instead uses the Java implementation provided by http://geotools.org/[Geotools].
+Geotools provides Java equivalent classes for OGC Filter XML elements.
+Geotools originally provided the standard Java classes for the OGC Filter Encoding 1.0 under the package name `org.opengis.filter`.
+The same package name is used today and is currently used by ${branding}. 
+Java developers do not parse or view the XML representation of a Filter in ${branding}. Instead, developers use only the Java objects to complete query tasks.
+
+Note that the `ddf.catalog.operation.Query` interface extends the `org.opengis.filter.Filter` interface, which means that a Query object is an OGC Java Filter with Query Options.
+
+.A Query is an OGC Filter
+[source,java]
+----
+public interface Query extends Filter
+----
+
+==== `FilterBuilder` API
+
+To abstract developers from the complexities of working with the Filter interface directly and implementing the ${branding} Profile of the Filter specification, the ${ddf-catalog} includes an API, primarily in `${ddf-branding}.filter`, to build Filters using a fluent API.
```

Comment:
The beginning of this sentence is odd : `To abstract developers from the complexities`.  Maybe just `Due to the complexities`?